externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkTBinaryFrontendService.scala

```diff
@@ -75,6 +64,13 @@ class SparkTBinaryFrontendService(
       val respConfiguration = new java.util.HashMap[String, String]()
       respConfiguration.put("kyuubi.engine.id", sc.applicationId)
 
+      if (req.getConfiguration != null) {
```

Comment:
Now the token is updated using serialized credentials in `openSession`, but the initial engine initialization sql (`kyuubi.engine.initialize.sql`) may fail to execute. 
For example, in master local mode, the engine does not have a Hive delegation token.

Reply:
cc @yaooqinn

Reply:
we can also pass token via startup spark configuations

Reply:
@turboFei seems this comment has not been addressed, can you look at it ?

Reply:
ok